powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthServiceConfiguration.java

```diff
@@ -141,6 +141,14 @@ public class PowerAuthServiceConfiguration {
     @Max(64)
     private long signatureValidationLookahead;
 
+    /**
+     * When validating the offline (or decimalized) signature, how many digits should a factor-related component have.
+     */
+    @Value("${powerauth.service.crypto.decimalComponentLength}")
```

Comment:
What about to rename the config option to `.offlineSIgnatureComponentLength` to avoid naming conflict with still supported decimalized online signatures?